challenge/docker-entrypoint.d/10_start_vm.sh

```diff
@@ -1,3 +1,14 @@
 #!/bin/sh
 
-find /challenge -name '*.ko' -exec false {} + || vm start
+if ! find /challenge -name '*.ko' -exec false {} +
+then
+  vm start
+else
+  chmod -s "$(which vm)"
+fi
+
+if [[ -f /challenge/challenge.exe ]]; then
```

Comment:
Can we do the same '*.exe' style, to allow more flexibility in challenge names?

Reply:
Even though `ChallengeProxy` can only run `challenge.exe`?